javascripts/background_scripts/embeded.js

```diff
@@ -0,0 +1,38 @@
+/*global chrome */
+/*global Privly */
+Privly.message.addListener(function (data, sendResponse) {
+  if (typeof data.action !== 'string') {
+    return;
+  }
+
+  if (data.action.indexOf('embeded/contentScript/') === 0) {
```

Comment:
There are several cases here and it is taking me too long to figure out what they are. You need more comments.